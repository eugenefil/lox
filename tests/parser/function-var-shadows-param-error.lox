fn f(x) { var x; }
