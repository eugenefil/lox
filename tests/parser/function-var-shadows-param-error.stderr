error: variable 'x' shadows function parameter of the same name
 --> $DIR/function-var-shadows-param-error.lox:1:11
  |
1 | fn f(x) { var x; }
  |           ^^^^^^
